### DISCUSSION TITLE
Responsive design in .net Maui

### DISCUSSION THREAD

**Discussion by math242j** [dotnet/maui#16854] [General]
I have an application where i until now how created and tested the layout/design for windows on a desktop version. Now i want to customize the layout to a mobile app. How is the best way?
My application layout is generel in Grids with 3 columns, and I was thinking that the columns should just be below each others?

**Comment by Kristoffeh** [comment]
I'm not too sure what you're asking for here.

If you're trying to get 3 more rows below your existing columns, those will be called rows. Just add rows the same way you added columns.
